README.md

```diff
@@ -58,12 +58,36 @@ Of those, this plugin currently uses:
 
 In addition to the above standardised tags, this plugin also adds:
 
+  - `component` (sent to Zipkin as "lc", for "local component")
   - `kong.api` (deprecated)
   - `kong.consumer`
   - `kong.credential`
   - `kong.node.id`
   - `kong.route`
   - `kong.service`
-  - `kong.balancer.try`
-  - `kong.balancer.state`: see [here](https://github.com/openresty/lua-resty-core/blob/master/lib/ngx/balancer.md#get_last_failure) for possible values
-  - `kong.balancer.code`
+  - `kong.try_[n].ipv4`
+  - `kong.try_[n].ipv6`
+  - `kong.try_[n].peer.port`
+  - `kong.try_[n].state`
+  - `kong.try_[n].code`
+
+The `kong.try_[n]` tags encode balancer tries, where `[n]` is a load balancing attempt number.
+See [here](https://github.com/openresty/lua-resty-core/blob/master/lib/ngx/balancer.md#get_last_failure) for possible values.
+
+## Logs
+
+Logs are used to encode the begin and end of every kong phase.
+
+  - `kong.rewrite`, `start` / `finish`, `<timestamp>`
+  - `kong.access`, `start` / `finish`, `<timestamp>`
+  - `kong.preread`, `start` / `finish`, `<timestamp>`
+  - `kong.header_filter`, `start` / `finish`, `<timestamp>`
+  - `kong.body_filter`, `start` / `finish`, `<timestamp>`
+  - `kong.try_[n]`, `start` / `finish`, `<timestamp>`
```

Comment:
the try_n annotation is a nice thing for the top-level span, but each load balancer try should be a sub-span with a different span ID, as you don't want to repeat the same span ID in multiple places. each LB try is a different http request, in other words. Also the tag encoding will affect search in a bad way whereas kong.try_1 will not.

Reply:
Thanks for the feedback, I have reverted everything involving the `try_[n]` tags. Each load balancer try is a new span.